TransformatieRDF.md

```diff
@@ -869,23 +868,23 @@ De tweede delete-insert query is een "opruimquery": aangezien we zijn begonnen m
 ### Transformatie: Domein
 > Het eigen IM.
 
-Een `mim:Domein` wordt omgezet naar een `owl:Ontology`, waarbij de naam en locatie eigenschappen worden gebruikt om de URI tot stand te brengen.
+Een `mim:Domein` wordt omgezet naar een `owl:Ontology`.
 
 ### Transformatie: Extern
 > Een groepering van constructies die een externe instantie beheert en beschikbaar stelt aan een informatiemodel en die in het informatiemodel ongewijzigd gebruikt worden.
 
-Een `mim:Extern` wordt omgezet naar een `owl:Ontology`, waarbij de naam en locatie eigenschappen worden gebruikt om de URI tot stand te brengen. Bovendien wordt een `owl:imports` aangelegd tussen de package van het type `mim:Domein` en deze externe package.
+Een `mim:Extern` wordt omgezet naar een `owl:Ontology`. Bovendien wordt een `owl:imports` aangelegd tussen de package van het type `mim:Domein` en deze externe package.
 
 De gedachte hierachter, is dat een externe package letterlijk is overgenomen vanuit een extern model, en hier aanvullend is meegenomen. Feitelijk zal de inhoud dus identiek moeten zijn aan het model op de betreffende externe locatie.
 
 ### Transformatie: View
 > Een groepering van objecttypen die gespecificeerd zijn in een extern informatiemodel en vanuit het perspectief van het eigen informatiemodel inzicht geeft welke gegevens van deze objecttypen relevant zijn binnen het eigen informatiemodel.
 
-Een `mim:View` wordt omgezet naar een `owl:Ontology`, waarbij de naam wordt gebruikt om de URI tot stand te brengen. Daarbij geldt dat voor de locatie, de locatie wordt overgenomen uit de package van het type `mim:Domein`. Bovendien wordt een `owl:imports` relatie gelegd tussen de package van het type domein en deze view-package, EN er wordt een `owl:imports` gelegd van de view-package naar een externe locatie, op basis van de locatie bij deze view package.
+Een `mim:View` wordt omgezet naar een `owl:Ontology`. Daarbij geldt dat voor de locatie, de locatie wordt overgenomen uit de package van het type `mim:Domein`. Bovendien wordt een `owl:imports` relatie gelegd tussen de package van het type domein en deze view-package, EN er wordt een `owl:imports` gelegd van de view-package naar een externe locatie, op basis van de locatie bij deze view package.
```

Comment:
hmm dit lijkt een issue in de bestaande spec. Een View hoeft geen onderdeel te zijn van een een Domein package. Wellicht kunnen we dit direct fixen.